src/modules/blockchainApplication/explore/components/BlockchainApplicationList/BlockchainApplicationList.js

```diff
@@ -1,61 +1,50 @@
-import React, {
-  useCallback, useMemo, useRef, useState,
-} from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import BoxHeader from 'src/theme/box/header';
-import Box from 'src/theme/box';
-import BoxContent from 'src/theme/box/content';
 import grid from 'flexboxgrid/dist/flexboxgrid.css';
-import Table from 'src/theme/table';
+import BoxHeader from '@theme/box/header';
+import Box from '@theme/box';
+import BoxContent from '@theme/box/content';
+import { QueryTable } from '@theme/QueryTable';
 import { Input } from 'src/theme';
 import Icon from 'src/theme/Icon';
+import { useFilter } from '@common/hooks';
+import { useBlockchainApplicationExplore } from '../../hooks/queries/useBlockchainApplicationExplore';
 import BlockchainApplicationRow from '../BlockchainApplicationRow';
 import header from './BlockchainApplicationListHeaderMap';
 import styles from './BlockchainApplicationList.css';
-import { BLOCKCHAIN_APPLICATION_LIST_LIMIT } from '../../const/constants';
 
-const BlockchainApplicationList = ({
-  applications,
-  applyFilters,
-  filters,
-}) => {
+const BlockchainApplicationList = () => {
   const [searchValue, setSearchValue] = useState('');
+  const [params, setParams] = useState({});
+  const { filters, applyFilters } = useFilter();
   const debounceTimeout = useRef(null);
   const { t } = useTranslation();
 
-  const canLoadMore = useMemo(() =>
-    (applications.meta
-      ? applications.data.length < applications.meta.total : false), [applications]);
+  useEffect(() => {
+    setParams(filters);
+  }, [filters]);
```

Comment:
is there a need to have another state that holds filter params useFilter already and hold state values of the filter. I think this is duplication. Already filter values can be used

Reply:
@eniolam1000752 Fixed